documentation/docs/10-adapters.md

```diff
@@ -31,6 +31,15 @@ export default {
 };
 ```
 
+Some adapters may modify your project's `.gitignore`, `.prettierignore` and `.eslintignore` to include their build output. In case you don't want those patterns included you can comment them out: 
```

Comment:
```suggestion
Some adapters may modify your project's `.gitignore` to include their build output. In case you don't want those patterns included you can comment them out: 
```